atlas-core/src/main/java/io/atlasmap/core/DefaultAtlasConversionService.java

```diff
@@ -65,18 +70,17 @@ public static DefaultAtlasConversionService getInstance() {
     }
 
     @Override
-    @SuppressWarnings("rawtypes")
-    public Optional<AtlasConverter> findMatchingConverter(FieldType source, FieldType target) {
+    public Optional<AtlasConverter<?>> findMatchingConverter(FieldType source, FieldType target) {
 
-        Optional<AtlasConverter> primitiveConverter = Optional.empty();
-        Optional<AtlasConverter> customConverter = Optional.empty();
+        Optional<AtlasConverter<?>> primitiveConverter = Optional.empty();
+        Optional<AtlasConverter<?>> customConverter = Optional.empty();
```

Comment:
These were there before, but the declarations and definitions above are unnecessary since both variables are redefined below in lines 81 and 85.   The declarations should probably be moved there.